### PR TITLE
chore: don't show operators when clicked

### PR DESCRIPTION
**Pull request by cealachoc**

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -7,10 +7,14 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+  const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
+
+  const onButtonClick = (val) => setValue(operators.includes(val) ? '' : val)
+
+  // function onButtonClick(val) {
```
Comment (krishna-acondy) [inline src/components/Calculator.js]: Could we not get rid of the commented out code?

File: src/components/Calculator.js [hunk 2]
```diff
@@ -7,10 +7,14 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+  const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
+
+  const onButtonClick = (val) => setValue(operators.includes(val) ? '' : val)
```
Comment (krishna-acondy) [inline src/components/Calculator.js]: This is setting the value to `''` if `val` is an operator. That means the value will disappear from the screen whenever you click an operator. Is that what happens on an actual calculator?

File: src/components/Calculator.js (hunk 1 above)
Comment (cealachoc) [reply src/components/Calculator.js]: Oh whoops! I thought i did get rid of it

File: src/components/Calculator.js [hunk 3]
```diff
@@ -7,10 +7,9 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+  const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
 
-  function onButtonClick(val) {
-    setValue(val)
-  }
+  const onButtonClick = (val) => setValue(prevValue => operators.includes(val) ? prevValue : val)
```
Comment (krishna-acondy) [inline src/components/Calculator.js]: This works, but we'd be better off not even calling `setValue` if `val` is an operator.
So something like:
```
  const onButtonClick = (val) => {
    if (!operators.includes(val) {
      setVal(val)
    }
  }

File: src/components/Calculator.js [hunk 4]
```diff
@@ -7,9 +7,14 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+  const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
 
-  function onButtonClick(val) {
-    setValue(val)
+  //const onButtonClick = (val) => setValue(prevValue => operators.includes(val) ? prevValue : val)
```
Comment (krishna-acondy) [inline src/components/Calculator.js]: This can go.

**Pull request merged**